ios/FluentUI/Popup Menu/PopupMenuController.swift

```diff
@@ -88,6 +88,7 @@ open class PopupMenuController: DrawerController {
                 if headerItem.subtitle == nil {
```

Comment:
What happens when the subtitle isn't nil? It looks like it should default to zero in that case, and I'm not sure there's a concise way to say "titleNumberOfLinesAsLongAsNoSubtitle", but we should probably at least add a documentation comment that warns the user that it doesn't apply when there is a subtitle, or that it wouldn't reset to 0 if they later add one.

Reply:
Oh good point. So it looks like when the subtitle isn't nil, then it uses a PopupMenuItemCell, subclassed from TableViewCell, instead of a UIView. In that case we can apply the titleNumberOfLines to the subtitle use case too